src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/PackageFeeds/RecommenderPackageFeed.cs

```diff
@@ -8,12 +8,12 @@
 using System.Threading.Tasks;
 using Microsoft.DataAI.NuGetRecommender.Contracts;
 using Microsoft.VisualStudio.Shell;
-using Microsoft.VisualStudio.Threading;
+using NuGet.Common;
 using NuGet.Packaging.Core;
 using NuGet.Protocol.Core.Types;
 using NuGet.VisualStudio;
 using NuGet.VisualStudio.Internal.Contracts;
-using NuGet.VisualStudio.Telemetry;
+using AsyncLazyNuGetRecommender = Microsoft.VisualStudio.Threading.AsyncLazy<Microsoft.DataAI.NuGetRecommender.Contracts.IVsNuGetPackageRecommender>;
```

Comment:
Normally, this would be the generic async lazy and not the one with the type.

Any particular reason why we have it this way?

I just searched and turns out we have 2 of these instances, which I don't love either, but trying to understand the motivation.

Reply:
Somehow it doesn't let me do generic async lazy declaration.
![image](https://user-images.githubusercontent.com/8766776/174358208-1c6f2438-cf86-4ad9-8009-b2e1dab18f01.png)

Also have seen we're already in other places:
https://github.com/NuGet/NuGet.Client/blob/e9637ed865687c2153da78c6c6385f010277329c/src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetPackageSearchService.cs#L35-L36

Reply:
I'm ok if you do what you linked. 

Up to you, I'm comfortable with either variant.
Let me know which one you want and I'll approve.

Reply:
I want to keep current variant, at least next time we can replace 1 place.